ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -410,6 +410,40 @@ def test_numpy_standard_gamma(
     )
 
 
+@handle_frontend_test(
+    fn_tree="numpy.random.standard_t",
+    df=st.floats(min_value=0, max_value=20, exclude_min=True),
```

Comment:
Maybe get integers here instead of floats for `df`

Reply:
I had it as `integers` initially, but the documentation of `numpy.random.standard_t` says that `df` is a `float`. Do you still want me to change that to `integers`?

Reply:
This is okay then, unless its causing failures elsewhere